etc/ssdlc.md

```diff
@@ -0,0 +1,48 @@
+# MongoDB C Driver SSDLC Compliance Report
+
+<!--
+This document is intended for distribution with release archives of the MongoDB
+C Driver. It wil be copied into the release archive as ssdlc_compliance_report.md
+-->
+
+## Release Creator
+
+For information on the release creator, refer to the
+[C/C++ Release Info Spreadsheet](https://docs.google.com/spreadsheets/d/1yHfGmDnbA5-Qt8FX4tKWC5xk9AhzYZx1SKF4AD36ecY)
+(internal link).
+
+## Process Document
+
+<!-- DRIVERS-2892: replace with link to public-facing document once available. -->
+Not yet available.
+
+## Third-Party Dependencies and Vulnerabilities
+
+The tracking of security information for bundled third-party dependencies is
+performed using Snyk. Refer to the `etc/third_party_vulnerabilities.md` file
+within the repository or the release archive for vulnerability information known
+at the time of a release. Full bundled dependency information is available in
+the `cyclonedx.sbom.json` file within the release archive.
+
+## Static Analysis Findings
+
+Refer to the
+[SSDLC Static Analysis Reports](https://drive.google.com/drive/folders/17bjBnQ3mhEXvs6IK1rrTphJr0CUO2qZh)
+folder (internal) for release-specific reports. Available as-needed from the
+MongoDB C Driver team.
+
+## Security Testing Report
+
+Refer to the
+[Driver Security Testing Summary](https://docs.google.com/document/d/1y2K_RY4GZVXpQvv4JH_35mSzFRTawNJ3mibpvSBU8H0)
+document (internal). Available as-needed from the MongoDB C Driver team.
+
```

Comment:
```suggestion
<!-- TODO: replace with link to Evergreen waterfall build for tagged commit. -->
Refer to the [Evergreen waterfall build](TODO) (internal) for test results.
```

I added the link to Evergreen in the [libmongocrypt Security Testing Summary](https://docs.google.com/document/d/1dc7uvBzu3okAIsA8LSW5sVQGkYIvwpBVdg5v4wb4c4s/edit#heading=h.7i6v0c6gf2w4). Since this links to the common drivers document, consider adding a link to the Evergreen waterfall build here to satisfy the [requirement noted in slack](https://mongodb.slack.com/archives/C06F31ZEKU1/p1718920120640689?thread_ts=1718047795.692869&cid=C06F31ZEKU1).

Reply:
This is annoyingly circular, since the waterfall build URL for the release depends on the commit hash of the project, which depends on the content of this file, which will need to contain a link to the waterfall build for the release, which won't be ready until this file is committed for the release.

Alternatively, this file can be partially templated and have the URL inserted at the last moment by `+release-archive`. IIUC, the waterfall will eventually have a build for the commit of the Git tag?

Reply:
Linking directly to the build corresponding to the release commit seems like an unreasonable requirement when the compliance report is meant to be a part of the release itself.

imo linking to a dedicated Google Doc as done for libmongocrypt, which may be updated post-release with the inclusion of a link to the mainline commit build, is the best intermediate solution for this requirement.

Reply:
Unfortunately the document linked to for C driver is intended to be the shared document for all drivers, so everyone would be piling their links together in the file all of the time.

I've found a solution that inserts the link as part of the release archive automatically. (See e18170e)

Reply:
> the shared document for all drivers

I didn't mean the Driver Security Testing Summary (STS) document, I meant the [libmongocrypt-specific STS document](https://docs.google.com/document/d/1dc7uvBzu3okAIsA8LSW5sVQGkYIvwpBVdg5v4wb4c4s/edit#heading=h.7i6v0c6gf2w4) which defers to the Drivers STS document. The libmongocrypt STS document contains a link to the mainline commit build as proposed for its `1.10.1` release.